src/Compilers/VisualBasic/Portable/Binding/Binder_Operators.vb

```diff
@@ -212,6 +212,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 diagnostics = New DiagnosticBag()
             End If
 
+            Dim NoOverflowChecks As Boolean = CheckOverflow
```

Comment:
Need to rename `NoOverflowChecks` to `HasOverflowChecks` as it better reflects its intention.